sql/moz-fx-data-shared-prod/fenix_derived/attributable_clients_v1/query.sql

```diff
@@ -53,28 +53,15 @@ first_seen AS (
 ),
 adjust_client AS (
   SELECT
-    client_info.client_id AS client_id,
-    ARRAY_AGG(metrics.string.first_session_network ORDER BY submission_timestamp)[
-      SAFE_OFFSET(0)
-    ] AS adjust_network,
-    ARRAY_AGG(metrics.string.first_session_adgroup ORDER BY submission_timestamp)[
-      SAFE_OFFSET(0)
-    ] AS adjust_adgroup,
-    ARRAY_AGG(metrics.string.first_session_campaign ORDER BY submission_timestamp)[
-      SAFE_OFFSET(0)
-    ] AS adjust_campaign,
-    ARRAY_AGG(metrics.string.first_session_creative ORDER BY submission_timestamp)[
-      SAFE_OFFSET(0)
-    ] AS adjust_creative,
-    MIN(DATE(submission_timestamp)) AS first_session_date
+    client_id,
+    adjust_network,
+    adjust_adgroup,
```

Comment:
```suggestion
    adjust_ad_group AS adjust_adgroup,
```

Reply:
Or even better, could we update the column name to use the same naming in both tables? Would be easier to find & query data.